prepare

```diff
@@ -41,28 +41,6 @@ pull_snap() {
         print_error "Could not set up Snap! git submodule.\nPlease make sure that you have access to the Internet and git is installed in your system."
         exit 1
     fi
-    cd src/snap
-    git checkout master
-    if ! git pull; then
-        print_error "Could not pull latest Snap! version from Git.\nPlease make sure that you have access to the Internet and git is installed in your system."
-        cd ../..
-        exit 1
-    fi
```

Comment:
But `./prepare --snap` is supposed to pull the latest changes from Snap<i>!</i>/master, because that's where the official repo lays, that's the whole point of having a `dev` and a `master` branch in the Snap<i>!</i> repo now! :)

```
%./prepare --help
Usage: ./prepare [options]

    --snap              Pulls the latest Snap! version.
```